spec/identity_doc_auth/acuant/request_spec.rb

```diff
@@ -152,5 +152,58 @@
         expect(response.exception).to be_a(Faraday::ConnectionFailed)
       end
     end
+
+    context 'when the request resolves with a handled http error status' do
+      before do
+        allow(subject).to receive(:handle_http_response) do |http_response|
+          http_response
+        end
+      end
+
+      def shared_expects(response)
+        expect(response.success?).to eq(false)
+        expect(response.exception).to be_kind_of(IdentityDocAuth::RequestError)
+        expect(response.exception.message).not_to be_empty
+      end
```

Comment:
A few ideas:
1. can we name this something clearer? 
```suggestion
      def expect_failed_response(response)
        expect(response.success?).to eq(false)
        expect(response.exception).to be_kind_of(IdentityDocAuth::RequestError)
        expect(response.exception.message).not_to be_empty
      end
```
2. RSpec also has [shared examples](https://relishapp.com/rspec/rspec-core/docs/example-groups/shared-examples) maybe we do that instead? Would take some more refactoring, moving the webmock to a `before` in a `context` but otherwise same same. May not be worth it

Reply:
ah, I didn't know about that piece of RSpec. I'll look into it.